components/ParliamentaryGroupCard/index.js

```diff
@@ -0,0 +1,39 @@
+import * as CUI from '@chakra-ui/react';
+
+const ParliamentaryGroupCard = ({
+  logoParty = 'https://cdn.mercadonegro.pe/wp-content/uploads/2020/01/congreso-2020-alianza-para-el-progreso-1024x1024.png',
+  nameParty = 'Alianza por el Progreso',
+  members = '12',
+}) => {
+  return (
+    <CUI.Flex
+      direction="column"
+      maxW="44"
+      minH="15.25rem"
+      px="18.5"
+      pt="1.25rem"
+      pb="6"
```

Comment:
Tratemos de ajustar los spacings en base 4 para no lidiar con decimales e usar las medidas de CUI antes de rem
```suggestion
      px="5"
      pt="6"
      pb="4"
```